src/app/resources/browser/assets/css/darkmode.css

```diff
@@ -1,11 +1,28 @@
 /* Dark mode (see #466). */
 /* TODO: Check if we need -webkit-filter for old Qt WebKit. */
+
+body {
+    background-color: white !important;
+    transition-duration: 0s !important;
```

Comment:
@kaygdev why is this needed?

Reply:
As far as I remember, a Docset still had a light background and was unreadable, but I could no longer reproduce this.
The second line seems to be an artifact from when I made the changes just for me. It suppressed an annoying animation in the Qt docset (highlight bright), I found this very annoying. Apparently that doesn't happen anymore either. I think the lines can be removed.

Reply:
Thanks for the quick response!

Reply:
Oh, I noticed the issue with Qt docset, probably that should be fixed in the docset, and not by us.